Tests/SubprocessTests/SubprocessTests+Darwin.swift

```diff
@@ -26,6 +26,19 @@ import Testing
 // MARK: PlatformOptions Tests
 @Suite(.serialized)
 struct SubprocessDarwinTests {
+    @Test func testSubprocessPlatformOptionsPreExecProcessAction() async throws {
```

Comment:
Could you also add a test to test setting both to make sure both works at the same time?

Reply:
Done